q2_vsearch/tests/test_cluster_features.py

```diff
@@ -696,6 +713,57 @@ def test_skip_denovo(self):
         exp_ref_seqs = _read_seqs(self.ref_sequences)
         self.assertEqual(obs_ref_seqs, exp_ref_seqs)
 
+    def test_skip_closed_reference(self):
+        # feature1 and feature3 clusters into r1 and feature2 and feature4
+        # clusters into r2 during closed-ref clustering; no unclustered
+        # features so de-novo clustering is skipped.
```

Comment:
Looks like copy/paste error with this comment.

Reply:
Oops, I think I was reviewing a single commit, sorry! Ignore this if it's already fixed.